src/scripts.js

```diff
@@ -50,38 +49,53 @@ showPantryRecipes.addEventListener("click", findCheckedPantryBoxes);
 searchForm.addEventListener("submit", pressEnterSearch);
 
 //-----fetch request---------
+function checkData() {
+  Promise.all([getUsers(), getIngredients(), getRecipes()])
+  .then(data => loadPageInfo(data))
+  .catch(error => console.log(error))
+}
+
 function getUsers() {
-  fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData')
+  return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData')
     .then(response => response.json())
-    .then(data => {users = data.wcUsersData, generateUser(users)})
+    .then(data => data.wcUsersData)
     .catch(error => console.log(error))
 }
 
 function getIngredients() {
-  fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/ingredients/ingredientsData')
+  return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/ingredients/ingredientsData')
     .then(response => response.json())
-    .then(data => {ingredientsData = data.ingredientsData, findPantryInfo(ingredientsData)})
+    .then(data => data.ingredientsData)
     .catch(error => console.log(error))
 }
 
 function getRecipes() {
-  fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/recipes/recipeData')
+  return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/recipes/recipeData')
     .then(response => response.json())
-    .then(data => {recipeData = data.recipeData, findTags(recipeData), createCards(recipeData)})
+    .then(data => data.recipeData)
     .catch(error => console.log(error))
 }
 
+function loadPageInfo(allData) {
+  allUsersData = allData[0]
+  ingredientsData = allData[1]
+  recipeData = allData[2]
+  generateUser(allUsersData)
+  findPantryInfo(ingredientsData)
+  findTags(recipeData)
+  createCards(recipeData)
+}
```

Comment:
Super pumped about the success of Promise. all and using this function to load the data. Way to go!